sleap_io/io/slp.py

```diff
@@ -248,16 +248,18 @@ def read_labels(labels_path: str) -> Labels:
     )
 
     frames = read_hdf5_dataset(labels_path, "frames")
-    lfs = []
-    for frame_id, video_id, frame_idx, instance_id_start, instance_id_end in frames:
-        lfs.append(
+    labeled_frames = []
+    for _, video_id, frame_idx, instance_id_start, instance_id_end in frames:
+        labeled_frames.append(
             LabeledFrame(
                 video=videos[video_id],
                 frame_idx=frame_idx,
                 instances=instances[instance_id_start:instance_id_end],
             )
         )
 
-    labels = Labels(lfs)
+    labels = Labels(
+        labeled_frames=labeled_frames, videos=videos, skeletons=skeletons, tracks=tracks
+    )
```

Comment:
Not strictly necessary since `Labels` discovers these automatically post construction [here](https://github.com/talmolab/sleap-io/blob/03fee325ad748109287eade72f975425e337d649/sleap_io/model/labels.py#L49-L60).

Reply:
Thanks, I think I did not see it happening because I started looking at your code before #6.